src/module/actor/sheets/SR5BaseActorSheet.ts

```diff
@@ -517,6 +509,15 @@ export class SR5BaseActorSheet extends ActorSheet {
                 // don't process anything else since we handled the drop
                 return;
             }
+            if (data.type === 'Actor' && data.uuid !== this.actor.uuid) {
+                const actor = await fromUuid(data.uuid) as SR5Actor;
+                const itemData = {
+                    name: actor.name ?? `${game.i18n.localize('SR5.New')} ${Helpers.label(game.i18n.localize(SR5.itemTypes['contact']))}`,
```

Comment:
Helpers.label is some legacy thing I suspect was used for skills and for headline capitalization, which the contact label already is by itself. I advice against using it when having no specific need for it (and removing it otherwise).

@smilligan93 Do you recall it's purpose? Maybe we should just remove it, as it did cause some minor bugs this year.